e2e-tests/gef/cypress/e2e/application-activities-AIN.spec.js

```diff
@@ -44,9 +45,8 @@ context('Submit AIN deal and check portalActivities', () => {
 
       // Make the deal an Automatic Inclusion Notice
       applicationDetails.automaticCoverDetailsLink().click();
-      automaticCover.automaticCoverTerm().each(($el) => {
-        $el.find('[data-cy="automatic-cover-true"]').trigger('click');
-      });
+      cy.automaticEligibilityCriteria();
+      cy.automaticEligibilityCriteria();
```

Comment:
```suggestion
      cy.automaticEligibilityCriteria();
```
I don't think we meant to have this twice